modules/gapi/include/opencv2/gapi/render.hpp

```diff
@@ -0,0 +1,53 @@
+#ifndef OPENCV_GAPI_RENDER_HPP
+#define OPENCV_GAPI_RENDER_HPP
+
+#include <string>
+#include <vector>
+
+#include <opencv2/gapi/opencv_includes.hpp>
+#include <opencv2/gapi/util/variant.hpp>
+#include <opencv2/gapi/own/exports.hpp>
+
+namespace cv
+{
+namespace gapi
+{
+
+struct TextEvent
+{
+    std::string text;
+    float x;
+    float y;
+    int ff;
+    double fs;
+    cv::Scalar color;
+    int thick;
+    int lt;
+    bool bottom_left_origin;
+};
+
+struct RectEvent
+{
+    float x;
```

Comment:
Again, may be it's better to use native OpenCV data structures?

Reply:
Fixed